app/workers/stat/update_worker.rb

```diff
@@ -0,0 +1,31 @@
+class Stat
+  class UpdateWorker
+    include Sidekiq::Worker
+
+    def perform(kind, user, library_entry_id)
+      library_entry = LibraryEntry.find(library_entry_id)
+      library_event = LibraryEvent.create_for(:updated, library_entry)
+
+      case kind
+      when :anime
+        # TODO: Change this before merging PR 201
+        Stat::AnimeActivityHistory.increment(user, library_event)
+        # special case checking if progress was increased or decreased
+        if progress > progress_was
```

Comment:
Where did these variables even come from

Reply:
oh god... Good catch, need to pass those in I think....